src/QTRunAction.cc

```diff
@@ -53,15 +52,15 @@ EGRunAction::EGRunAction(QTEventAction* eventAction, G4String name, G4int na)
   analysisManager->CreateNtupleDColumn("Vposy");
   analysisManager->CreateNtupleDColumn("Vposz");
   for (G4int i=0;i<nAntenna;++i) {
-    analysisManager->CreateNtupleTColumn(tvecname + std::to_string(i), fEventAction->GetTimeVec(i));
-    analysisManager->CreateNtupleTColumn(vvecname + std::to_string(i), fEventAction->GetVoltageVec(i));
+    analysisManager->CreateNtupleDColumn(tvecname + std::to_string(i));
```

Comment:
@yramachers I'm not convinced my "fixes" here are correct. It looks like you're trying to record a name-value pair here? Is that intentional - `CreateNtupleTColumn` doesn't appear to be a member of any of the Analysis Manager classes, and it's not used in any of the examples. I've dug a bit into the source code of `G4NtupleBookingManager.cc` and it doesn't appear to match what you're doing here either?

Reply:
@TomGoffrey , I agree; it looks like the TColumn as a template column creation is a private method of the booking manager. Don't remember why I used it explicitly other than making sure that we need to store a full std::vector here in the column. It is possible to use the storage of values in the vector but that would be another loop over vector content. This ability to store a vector as single entry in a row was enabled in older versions, then disappeared, then apparently shows up again, confusing. 
Nevertheless, storing a full vector as a single row entry is extremely useful for the output file.
So, yes, DColumn will do but I think we need to use the booking manager method

G4int CreateNtupleDColumn(const G4String& name, std::vector<double>* vector);

which ought to set up storage of a std::vector.

Reply:
Thanks!

Reply:
Adding discussion at:
https://geant4-forum.web.cern.ch/t/filling-std-vector-type-ntuples-using-g4root/2118

for reference. See also recent FIXME comments.